net/tcp/tcp_recvwindow.c

```diff
@@ -163,7 +165,29 @@ uint32_t tcp_get_recvwindow(FAR struct net_driver_s *dev,
       tailroom = 0;
     }
 
+  /* Get the number of buffers that the NIC needs to reserved */
+
+  niob_reserved = 0;
+  iob = dev->d_iob;
+
+  while (iob != NULL)
```

Comment:
should we add iob function for counting?

Reply:
Done